packages/vm/examples/run-blockchain/index.ts

```diff
@@ -72,14 +71,15 @@ async function setupPreConditions(vm: VM, testData: any) {
 }
 
 async function setGenesisBlock(blockchain: any, common: Common) {
-  const header = new BlockHeader(testData.genesisBlockHeader, { common })
-  const genesisBlock = new Block([header.raw, [], []], { common })
-  await blockchain.putGenesis(genesisBlock)
+  const header = testData.genesisBlockHeader
+  const genesis = Block.genesis({ header }, { common })
```

Comment:
*scrolls down...*

*scrolls down...*

Phew, I really wasn't aware that there were so many references / usages of this genesis functionality. 😃